sensu/plugins/check-glance-store.py

```diff
@@ -15,6 +15,11 @@
 from iso8601 import iso8601
 from keystoneclient.v2_0 import client as ksclient
 
+def is_remote_image(image):
```

Comment:
perhaps reversing this logic and checking if it is a local image? I suspect glance would handle a ftp url as well.